.golangci.yml

```diff
@@ -39,6 +39,9 @@ linters-settings:
       mnd:
         # don't include the "operation" and "assign"
         checks: argument,case,condition,return
+        ignored-functions:
```

Comment:
Can you please explain in the commit message what was the error ans why it is ignored?

Reply:
Done